determine_result.php

```diff
@@ -0,0 +1,25 @@
+<?php
+    
+    function determineResult($bidPrice, $highestPrice, $bid_status)
+    // can only be shown on 1 page at a time
+    {
+        if ($bidPrice > $highestPrice AND $bid_status == 'Finished')
+        {
+            return 'You Won the bidding';
+        }
+        elseif ($bidPrice <= $highestPrice AND $bid_status == 'Finished')
+        {
+            return 'You lost the bidding';
+
+        }
+        elseif ($bid_status == 'Ongoing')
```

Comment:
@yly828 Feel free to remove $bid_status from your function since we only consider "Finished" bid from now on.